tests/js/spec/views/organizationDiscover/conditions/condition.spec.jsx

```diff
@@ -55,6 +56,15 @@ describe('Condition', function() {
       expect(options[0]).toEqual({value: 'col2 >', label: 'col2 >'});
     });
 
+    it('renders operator options for datetime column', function() {
+      const options = wrapper.instance().filterOptions([], 'col3');
+      expect(options).toHaveLength(4);
+      expect(options[0]).toEqual({value: 'col3 =', label: 'col3 ='});
```

Comment:
are there options for `>` and `<`?

Reply:
Good point, we should allow these operators too